EMS/core-bundle/src/Controller/UserController.php

```diff
@@ -307,4 +311,36 @@ private function userExist(User $user, string $action): bool
 
         return true;
     }
+
+    public function removeFromGroup(User $user, string $groupName): Response
+    {
+        if($user->getUserGroup() !== $groupName){
+            throw new \RuntimeException('Group name has to be equal to user group name');
+        }
+        $user->setUserGroup(null);
+        $this->userService->updateUser($user);
+        $userGroup = $this->groupManager->getByItemName($groupName);
+        
+        if (!$userGroup instanceof EntityInterface) {
+            throw new EntityNotFoundException();
+        }
+        return $this->redirectToRoute(Routes::GROUP_EDIT, [
+            'group' => $userGroup->getId(),
+        ]);
+    }
+
+    public function addFromGroup(User $user, string $group): Response
+    {
+        $user->setUserGroup($group);
+        $this->userService->updateUser($user);
+        $userGroup = $this->groupManager->getByItemName($group);
+
+        if (!$userGroup instanceof EntityInterface) {
+            throw new EntityNotFoundException();
+        }
```

Comment:
```suggestion
        $userGroup = $this->groupManager->getByItemName($group);
        if (!$userGroup instanceof EntityInterface) {
            throw new EntityNotFoundException();
        }
        $this->userService->updateUser($user);
```
Test then update (not the opposite)

Reply:
ok